Scientific documentation for SRW v3.0.0 release (#257)
ufs-community/ccpp-physics: physics/docs/pdftxt/SMOKE_DUST.txt

/**
\page SMK_DUST SRW Smoke and Dust Scheme
\section des_smkdust Smoke Scheme Description

The SRW v3.0.0 release includes the smoke modeling capability. In the model, a single aerosol tracer (smoke)
represents primary fine particulate matter (PM2.5) from biomass burning (BB) emissions. The BB emissions are
estimated using the fire radiative energy data from the regional ABI and VIIRS Emissions (RAVE) product
(Li et al. 2022 \cite LI2022113237), which integrates fire radiative power (FRP) data from the Geostationary 
Operational Environmental Satellites (GOES-R ABI) and the Joint Polar Satellite System (JPSS VIIRS). The FRP data

> from RAVE is used to estimate the fire heat flux in SRW to simulate the fire plumerise in the model. The SRW application

Reviewer: ```suggestion
from RAVE is used to estimate the fire heat flux in SRW to simulate the fire plume rise in the model. The SRW application
```

|--------------------------------|-------------------------|---------------------|
|AEMFLX                          |Smoke Emissions          |kg/m^2/s             |
|WFIREPOT (or var discipline=2 master_table=2 parmcat=4 parm=26)      | hourly wildfire potential index     | unitless |
|COLMD: entire atmosphere (considered as a single layer):anl:chemical=Particulate Organic Matter Dry    | vertically integrated smoke  | kg/m^2   |
|MASSDEN:8 m above ground:anl:chemical=Particulate Organic Matter Dry:aerosol_size<2.5e-6  | near surface smoke  | kg/m^3     |
|MASSDEN:8 m above ground:1-2 hour ave fcst:chemical:Particulate Organic Matter Dry:aerosol_size<2.5e-06    | 1-h avg near surface smoke (PM2.5) | kg/m^3   |
| AOTK                                                                |aerosol optical depth                | unitless |                                        
\tableofcontents 

\section Fengsha_desp FENGSHA Dust Emission Scheme

Aerosols have both direct and indirect effects on meteorology, atmospheric chemistry, human health, and ultimately 
the global energy budget with dust being a major contributor to the atmospheric aerosol burden. A great effort has been

> made to characterized the sources and mobilization of dust, however, current models still show large uncertainty as the 

Reviewer: ```suggestion
made to characterize the sources and mobilization of dust, however, current models still show large uncertainty as the 
```

\f[
 Q = C \frac{\rho_{a}}{g}(R u_{*}^3) \left ( 1 - \frac{u_{*t}^2(R,H)}{u_{*}^2} \right ) \left ( 1 + \frac{u_{*t}(R,H)}{u_{*}} \right )

\f]

- \b Threshold \b Friction \b Velocity: Soil properties control the minimum wind force required to mobilize dust particles, refined by
Gillette's empirical findings.
- \b Vertical-to-Horizontal \b Flux \b Ratio: Derived from Marticorena and Bergametti (1995) \cite marticorena_and_bergametti_1995, this
calculation links friction velocity to emission processes.
- \b Drag \b Partition \b Correction: Based on MacKinnon et al. (2004) \cite MACKINNON2004103, these adjustments account for surface
roughness influences on dust emissions.
- \b Soil \b Moisture \b Effects: Incorporates corrections from Fecan and Shao methods to adjust for the impact of soil water content. 

FENGSHA is implemented in CCPP with modular subroutines for compatibility with other atmospheric components. Key routines include:
- dust_fengsha_mod::fengsha_drag: Calculates drag partition corrections, adjusting for surface roughness effects.
- dust_fengsha_mod::dustemissionfengsha: Computes dust emissions using inputs such as soil composition, wind velocity, and air density.

> - dust_fengsha_mod::moisturecorrectionfecan and dust_fengsha_mod::moisturecorrectionshao: Refine emission thresholds based on soil moisture conditions.

Reviewer: ```suggestion
- dust_fengsha_mod::moisturecorrectionfecan and dust_fengsha_mod::moisturecorrectionshao: Refines emission thresholds based on soil moisture conditions.
```

- dust_fengsha_mod::dustfluxv2hratiomb95: Estimates vertical-to-horizontal flux ratios using clay content and flux limits.

  

\section intra_smkdust Intraphysics Communication
 +  \ref arg_table_rrfs_smoke_wrapper_run 

\subsection fensha_intra_com FENGSHA Dust Emission scheme
The scheme facilitates seamless interaction with other physical processes, including aerosol microphysical and radiation models,

> ensuring consistency in atmospheric simulations. FENGSHA'S modular structure allows its outputs -binned surface emissions - to integrate

Reviewer: ```suggestion
ensuring consistency in atmospheric simulations. FENGSHA'S modular structure allows its outputs — binned surface emissions — to integrate
```

smoothly into broader forecasting frameworks.

\section gen_al_smkdust FENGSHA Dust Emission General Algorithm
-# Initialize dust emissions and prepare scaling factors
-# Ensure that dust emissions do not occur over water, forests, bedrock or snow and ice
-# Apply soil moisture corrections using Fecan \cite angeo-17-149-1999 or Shao \cite shao_2004 methods 
-# Compute threshold wind friction velocity, accounting for drag partition corrections
-# Calculate horizontal saltation flux and distribute emissions across particle size bins
-# Convert emissions to mass flux, ensuring compatibility with downstream components

FENGSHA is employed in operational systems like GEFS-Aerosol(Zhang et al. 2021 \cite zhang_etal_2021) and NAQFC (Huang et al. 2025, 

> Li et al., 2024), enhancing air quility and dust even predictions. It provides detailed insights into the interactions between

Reviewer: ```suggestion
Li et al., 2024), enhancing air quility and dust event predictions. It provides detailed insights into the interactions between
```
